domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1873,10 +1873,11 @@ public TotaledResults(RowSet<Row> set, Integer total) {
         replyHandler.handle(Future.failedFuture(prepareRes.cause()));
         return;
       }
-      PreparedStatement pq = prepareRes.result();
-      RowStream<Row> stream = pq.createStream(STREAM_GET_DEFAULT_CHUNK_SIZE, Tuple.tuple());
```

Comment:
Would it work if adding `stream.endHandler/exceptionHandler` to close `pq` here?

Reply:
No, doStreamRowResults will overwrite endHandler and exceptionHandler.